Lib/test/pickletester.py

```diff
@@ -1982,8 +1982,10 @@ def test_picklebuffer_error(self):
         pb = pickle.PickleBuffer(b"foobar")
         for proto in range(0, 5):
             with self.subTest(proto=proto):
-                with self.assertRaises(pickle.PickleError):
+                with self.assertRaises(pickle.PickleError) as cm:
                     self.dumps(pb, proto)
+                self.assertEqual(str(cm.exception),
+                    'PickleBuffer can only be pickled with protocol >= 5')
```

Comment:
```suggestion
                self.assertEqual(str(cm.exception),
                    			 'PickleBuffer can only be pickled with protocol >= 5')
```

Reply:
The line will be too long.

Reply:
Oh ok :( Maybe a temporary variable then? (otherwise, let's leave it like this)